config/initializers/dfe_analytics.rb

```diff
@@ -0,0 +1,57 @@
+# frozen_string_literal: true
+
+DfE::Analytics.configure do |config|
+  # Whether to log events instead of sending them to BigQuery.
+  #
+  # config.log_only = true
+
+  # Whether to use ActiveJob or dispatch events immediately.
+  #
+  # config.async = true
+
+  # Which ActiveJob queue to put events on
+  #
+  config.queue = :dfe_analytics
+
+  # The name of the BigQuery table we’re writing to.
+  #
+  # config.bigquery_table_name = ENV['BIGQUERY_TABLE_NAME']
+
+  # The name of the BigQuery project we’re writing to.
+  #
+  # config.bigquery_project_id = ENV['BIGQUERY_PROJECT_ID']
+
+  # The name of the BigQuery dataset we're writing to.
+  #
+  config.bigquery_dataset = ENV["BIGQUERY_DFE_ANALYTICS_DATASET"] || Rails.application.credentials.BIGQUERY_DFE_ANALYTICS_DATASET
+
+  # Service account JSON key for the BigQuery API. See
+  # https://cloud.google.com/bigquery/docs/authentication/service-account-file
+  #
+  config.bigquery_api_json_key = ENV["BIGQUERY_DFE_ANALYTICS_API_JSON_KEY"] || Rails.application.credentials.BIGQUERY_DFE_ANALYTICS_JSON_API_KEY
+
+  # Passed directly to the retries: option on the BigQuery client
+  #
+  # config.bigquery_retries = 3
```

Comment:
maybe we can set the retries to max 3 here, to avoid too many retries?

Reply:
This is the default https://github.com/DFE-Digital/dfe-analytics/blob/6d42dc06a849c659b6b44280c799420273f78223/config/locales/en.yml#L34